src/validators/model.rs

```diff
@@ -35,6 +36,11 @@ impl BuildValidator for ModelValidator {
         // models ignore the parent config and always use the config from this model
         let config: Option<&PyDict> = schema.get_as("config")?;
 
+        let field_default_required = match config {
+            Some(dict) => !matches!(dict.get_as("model_full")?, Some(false)),
```

Comment:
```suggestion
            Some(config_dict) => dict.get_as("model_full")?.unwrap_or(true),
```
Not that your code is wrong, just best to be consistent with styles across the codebase, and this is how I do it elsewhere.

Reply:
cleaner syntax